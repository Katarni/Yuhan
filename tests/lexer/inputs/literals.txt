6.6
7.0
4e3
4er
4e
"eee"
5.
'g'
'h"ddd' 
