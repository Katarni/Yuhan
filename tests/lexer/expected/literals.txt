float literal 6.6 1:1
float literal 7.0 2:1
exponential literal 4e3 3:1
another 4er 4:1
another 4e 5:1
string literal eee 6:1
another 5. 7:1
char literal g 8:1
another h"ddd 9:1
string literal  10:1
